specs/subresourceintegrity/spec.markdown

```diff
@@ -748,6 +748,21 @@ will likely be difficult to avoid (image's `naturalHeight` and
 `naturalWidth` for instance).
 </section><!-- /Security::cross-origin -->
 
+<section>
+### Behavior of chameleon resources
```

Comment:
I've also heard the term 'Polyglots' in [Polyglots: Crossing Origins by Crossing Formats](http://www.cse.chalmers.se/~d02pulse/thesis/) by Jonas Magazinius et al., but I am not sure which is more common